src/core_landice/mode_forward/mpas_li_thermal.F

```diff
@@ -2452,15 +2666,20 @@ subroutine enthalpy_matrix_elements(&
       real(kind=RKIND) :: dEnthalpy             ! enthalpy difference between adjacent layers
       real(kind=RKIND) :: dEnthalpyTemp         ! difference in temperature component of enthalpy between adjacent layers
       real(kind=RKIND) :: avgFactor             ! factor for averaging diffusivity, 0 <= avgFactor <= 1
+      real(kind=RKIND) :: tempDiffusivityFactor ! factor for estimating the diffusivity in temperate ice, normally it should < 1
 
       integer  :: k
 
       logical, parameter :: &
            harmonic_avg  = .false.  ! if true, take the harmonic average of diffusivity in adjacent layers
                                     ! if false, take the arithmetic average
 
+      tempDiffusivityFactor = 1.0e-5_RKIND
```

Comment:
This diffusivity factor is kind of an arbitrary choice. There is no knowledge of that in the temperate layer. In this paper "Comparison of thermodynamics solvers in the polythermal ice sheet model SICOPOLIS", Greve and Blatter used a value of 1e-6. I can't remember exactly why I choose 1e-5, but I am pretty sure it's the result after I made some comparisons. Because if its uncertainty, I am not sure if it's better to put it in the namelist as a constant. Perhaps we should do that, but also put some notes somewhere (about its arbitrary nature)?

Reply:
I recommend that you add this parameter as a namelist option in Registry.xml.  Make the default value the value used in the Kleiner experiment.  In the description in Registry, you can add information about the uncertainty and range of typical values if you like (although the fact that a parameter is in Registry implies that it does not have a definite value).  Also add the units there.

Reply:
Will do. Thanks.

Reply:
I add a new 'config_temp_diffusive_factor' in the advection section in Registry. I also added some comments.